core/src/main/java/org/stellar/anchor/config/Sep10Config.java

```diff
@@ -67,22 +67,18 @@ public interface Sep10Config {
   boolean isClientAttributionRequired();
 
   /**
-   * Get the list of allowed client domains if the client attribution is required.
+   * Get the list of allowed client domains.
    *
    * @return the list of allowed client domains.
    */
-  List<String> getClientAttributionAllowList();
+  List<String> getAllowedClientDomains();
```

Comment:
Isn't it a breaking change?

Reply:
This should works just fine. Only the function name changed, the name of the config field stays`clientAllowList`,

Reply:
How did you achieve that?

Reply:
Can you elaborate more on the question? We didn't touch the field `clientAllowList` in Sep10Config..

The `getClientAttributionAllowList` returns a list of allowed client domains (which is the intersection of `sep10config.clientAllowList` and `clientsConfig.clients`), since we are introducing a new function to return the name of same group of clients, it's better to make the naming

Reply:
Sorry didn't see SEP10 config is an interface